### PR TITLE
Integrate PPL endpoint with workbench

### PR DESCRIPTION
**Pull request by chloe-zh**
*Issue #, if available:*

*Description of changes:*

- enable ppl endpoint in workbench
- renamed folder `sql-workbench` to `workbench`
- renamed plugin name to query workbench

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/sql-workbench-release-workflow.yml [hunk 1]
```diff
@@ -46,13 +46,13 @@ jobs:
         with:
           timeout_minutes: 60
           max_attempts: 3
-          command: cd kibana/plugins/sql-workbench; yarn kbn bootstrap
+          command: cd kibana/plugins/workbench; yarn kbn bootstrap
```
Comment (joshuali925) [inline .github/workflows/sql-workbench-release-workflow.yml:49]: line 42 `mv sql-workbench kibana/plugins` should be `mv workbench kibana/plugins`

File: .github/workflows/sql-workbench-release-workflow.yml [hunk 2]
```diff
@@ -46,13 +46,13 @@ jobs:
...
 
       - name: Build Artifact
         run: |
-          cd kibana/plugins/sql-workbench
+          cd kibana/plugins/workbench
           yarn build
           artifact=`ls ./build/*.zip`
 
-          aws s3 cp $artifact s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/kibana-plugins/opendistro-sql-workbench/
+          aws s3 cp $artifact s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/kibana-plugins/opendistro-query-workbench/
```
Comment (joshuali925) [inline .github/workflows/sql-workbench-release-workflow.yml:57]: `index.js` and `package.json` still use `sql-workbench`, where does `query-workbench` come from?

**Review by joshuali925 — APPROVED** [review]
Code looks good to me, might need to update plugin name in configs and readme file

**Review by anirudha — APPROVED** [review]

File: .github/workflows/sql-workbench-release-workflow.yml (hunk 2 above)
Comment (chloe-zh) [reply .github/workflows/sql-workbench-release-workflow.yml:57]: refracted in index.js and package.json

File: .github/workflows/sql-workbench-release-workflow.yml (hunk 1 above)
Comment (chloe-zh) [reply .github/workflows/sql-workbench-release-workflow.yml:49]: Done, thanks!

**Pull request merged**